File: app/src/main/java/com/example/stud_informer_irkpo/models/ScheduleResponse.java
package com.example.stud_informer_irkpo.models;

import java.util.List;

public class ScheduleResponse {
    private String currentDate;
    private int currentWeekType;
    private String currentWeekName;
    private List<DaySchedule> items;

    public String getCurrentDate() {
        return currentDate;
    }

    public void setCurrentDate(String currentDate) {
        this.currentDate = currentDate;
    }

    public int getCurrentWeekType() {
        return currentWeekType;
    }

    public void setCurrentWeekType(int currentWeekType) {
        this.currentWeekType = currentWeekType;
    }

    public String getCurrentWeekName() {
        return currentWeekName;
    }

    public void setCurrentWeekName(String currentWeekName) {
        this.currentWeekName = currentWeekName;
    }

    public List<DaySchedule> getItems() {
        return items;
    }

    public void setItems(List<DaySchedule> items) {
        this.items = items;
    }
}

File: app/src/main/java/com/example/irkpo_management/models/ScheduleResponse.kt
package com.example.irkpo_management.models

class ScheduleResponse {
    @JvmField
    var currentDate: String? = null
    @JvmField
    var currentWeekType: Int = 0
    @JvmField
    var currentWeekName: String? = null
    @JvmField
    var items: MutableList<DaySchedule?>? = null
}
